nservicebus/architecture/scaling.md

```diff
@@ -1,7 +1,12 @@
 ---
 title: Scaling with NServiceBus
 summary: NServiceBus provides several options to scale out a system
-reviewed: 2019-02-08
+tags:
+ - scalability
+ - routing
+redirects:
+ - nservicebus/architecture/scaling
```

Comment:
Shouldn't this be `transports/scale-out`?

Reply:
Good catch!